src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -25,8 +25,8 @@ limitations under the License.
     <server init="false" tick="false">true</server>
     <description>This cluster provides an interface for launching content on a media player device such as a TV or Speaker.</description>
 
-    <attribute side="server" code="0x0000" define="APPLICATION_LAUNCHER_LIST"        type="ARRAY" entryType="int16u" length="254" reportable="true"  writable="false" optional="true">CatalogList</attribute>
-    <attribute side="server" code="0x0001" define="APPLICATION_LAUNCHER_CURRENT_APP" type="ApplicationEPStruct"      default="0x0" isNullable="true" writable="true"  optional="true">CurrentApp</attribute>
+    <attribute side="server" code="0x0000" define="APPLICATION_LAUNCHER_LIST"        type="ARRAY" entryType="int16u" reportable="true"  writable="false" optional="true">CatalogList</attribute>
+    <attribute side="server" code="0x0001" define="APPLICATION_LAUNCHER_CURRENT_APP" type="ApplicationEPStruct"      isNullable="true" writable="false"  optional="true">CurrentApp</attribute>
```

Comment:
This is a breaking change.  This might need to get changed in the spec instead, if we can't figure out a way to not have client APIs break on this.